draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -1193,11 +1227,24 @@ boolean _httpOnlyAllowed_, boolean _allowNonHostOnlyCookieForPublicSuffix_, and
 
 1. Insert _cookie_ into the user agent's cookie store.
 
-1. Remove all expired cookies from the user agent's cookie store.
+1. Return _cookie_.
+
+
+### Garbage Collect Cookies
+
+To **Garbage Collect Cookies** given a host _host_:
 
-1. Run Remove Excess Cookies for Host given _cookie_'s host.
+1. Let _expiredCookies_ be a list containing all expired cookies in the user agent's cookie store.
```

Comment:
Let's talk about references to cookies again.

Reply:
Done.